python/graphstorm/eval/evaluator.py

```diff
@@ -845,17 +837,15 @@ def compute_score(self, scores, train=False): # pylint:disable=unused-argument
             return_metrics[metric] = return_metric.item()
         return return_metrics
 
-    def evaluate(self, val_scores, test_scores, total_iters):
```

Comment:
Do not change the naming of val_scores and test_scores.

Reply:
Ok. Reverted.